core/web/resolver/mutation.go

```diff
@@ -552,18 +552,13 @@ func (r *Resolver) CreateNode(ctx context.Context, args struct {
 }
 
 func (r *Resolver) DeleteNode(ctx context.Context, args struct {
-	ID graphql.ID
+	Name string
```

Comment:
Wouldn't we want to mark this as `graphql.ID` for type consistency? We are returning the same as a GQL ID anyways, as part of these changes.